testpilot/frontend/static-src/app/views/landing-page.js

```diff
@@ -37,7 +37,9 @@ export default PageView.extend({
     }
 
     app.sendToGA('pageview', {
-      'dimension1': this.loggedIn
+      'dimension1': this.loggedIn,
```

Comment:
I'm not really sure about the 'dimension' thing either. I got it from the ga reqs doc https://docs.google.com/document/d/1SIOaZLHsMGNBNmAxphon4cxF_F07Q1ctBkU6gkP2SD0/edit?ts=56fc371c